controllers/changetierrequest/changetierrequest_controller_test.go

```diff
@@ -224,6 +227,22 @@ func TestChangeTierFailure(t *testing.T) {
 			AllUserAccountsHaveTier(murtest.DefaultNSTemplateTier)
 		AssertThatChangeTierRequestHasCondition(t, cl, changeTierRequest.Name, toBeComplete(), toBeDeletionError("unable to delete ChangeTierRequest object 'request-name': error"))
 	})
+
+	t.Run("the change will fail because there is no corresponding UserSignup found", func(t *testing.T) {
+		// given
+		mur := murtest.NewMasterUserRecord(t, "john")
+		changeTierRequest := newChangeTierRequest("john", "team")
+		teamTier := NewNSTemplateTier("team", "123team", "123clusterteam", "stage", "dev")
+		controller, request, cl := newController(t, changeTierRequest, config, mur, teamTier)
+
+		// when
+		_, err := controller.Reconcile(context.TODO(), request)
+
+		// then
+		require.Error(t, err)
+		assert.Equal(t, err.Error(), `failed to get corresponding UserSignup for MasterUserRecord with name 'john': MasterUserRecord is missing label 'toolchain.dev.openshift.com/owner'`)
+		AssertThatChangeTierRequestHasCondition(t, cl, changeTierRequest.Name, toBeNotComplete(`MasterUserRecord is missing label 'toolchain.dev.openshift.com/owner'`))
+	})
 }
```

Comment:
And what about the test which verifies that the deactivating state was dropped for the UserSignup?

Reply:
I'm not sure how I missed that. 🤦 Added in 5fda34e9a2ec5ebf433621ea20aaf7dd8ef0699c